.gitignore

```diff
@@ -11,11 +11,12 @@
 /python/build
 /python/dist
 /python/python-driver-*
-/python/ray/serve/generated
+/python/ray/serve/generated/*
 /thirdparty/pkg/
 /build/java
 .jar
 /dashboard/client/build
+.whl
```

Comment:
I have a different viewpoint. In this case, I exec `./python/build-wheel-macos.sh` in mine dev machine, it always create the folder. Another, the `.jar` is the same case which is ignored and friendly to developer. So I think should add `.whl` to gitignore, so that it won't bother newbies. 😝

Reply:
ok. could you update the PR description that this is about building the macos wheel? thanks.

Reply:
yeah